static_inverse_kinematics.py

```diff
@@ -63,21 +66,46 @@ def __init__(
         self.c3d_path_file = c3d_path_file
```

Comment:
Is it used elsewhere? If not delete the line.

Reply:
it is used to load the c3d